### PR TITLE
Fixes a bug where Rezadone instantly healed you of all clone damage

### PR DESCRIPTION
**Pull request by ghost**
# Rezadone now heals you over time very quickly instead of instantly
#### Changelog

:cl:
rscadd: Rezadone now obeys the laws of sanity.
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 1]
```diff
@@ -161,16 +161,16 @@
...
 	..()
 	. = 1
 
 /datum/reagent/medicine/rezadone/overdose_process(mob/living/M)
-	M.adjustToxLoss(1, 0)
-	M.Dizzy(5)
-	M.Jitter(5)
+	M.adjustToxLoss(3, 0)
+	M.Dizzy(10)
+	M.Jitter(10)
```
Comment (ShadowDeath6) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm:173]: undocumented and also unneeded changes

Comment (ghost) [reply code/modules/reagents/chemistry/reagents/medicine_reagents.dm:173]: you're making it easier to get

this makes it hurt

Comment (ShadowDeath6) [reply code/modules/reagents/chemistry/reagents/medicine_reagents.dm:173]: making it do 3x tox damage is really not needed considering any possible threat will make people just switch to the non-ODing 250u cryox with no threats and fast healing

**Comment by Notamaniac** [comment]
Pretty sure rezadone healing all clone damage was intentional since it was never used

**Comment by X-TheDark** [comment]
Yeah, this isn't a bug.

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm (hunk 1 above)
Comment (Super3222) [reply code/modules/reagents/chemistry/reagents/medicine_reagents.dm:173]: Include those tweaks in your changelog.

**Comment by ghost** [comment]
it's still dumb

**Comment by Super3222** [comment]
> Merging a PR with CONFIRMED undocumented changes that are actually important

@Cruix

**Comment by Cruix** [comment]
No one ever makes redazone and if they do they probably aren't going to overdose on it. 7 units will heal any amount of clone damage you could possibly have, and it takes 30 to overdose.

**Pull request merged**